src/js/modules/api/showData.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable prefer-const */
+/* eslint-disable no-unused-vars */
+import Popup from '../popup.js';
+import getLikes from './getLikes.js';
+import postLikes from './postLikes.js';
+
+class DisplayShows {
+  constructor() {
+    this.allItemsCounter = (numberShows) => {
+      let showsReturn = [];
+      if (numberShows.length > 0) {
+        numberShows.length = 9;
+        showsReturn = numberShows;
+      }
+      return showsReturn.length;
+    };
+  }
```

Comment:
Hello @VSeray and @mugishasam123, I see that you implemented here the counter, going deeply in this function you are setting directly the number "9" items listed, and the idea here it's to actually count the items no matter how many they are. I will set this function as optional, it will be great for your career as developers to really go a little further here and implement a real counter before merging, maybe you can use document.querySelectorAll to do this 😉

### OPTIONAL

- [ ] Update items counter to make it more professional.